src/actions/placeBid.ts

```diff
@@ -29,28 +32,37 @@ export interface PlaceBidActionParameters {
 export const placeBid = async (
   params: PlaceBidActionParameters
 ): Promise<void> => {
+  logger.trace(`Calling to place a bid by user ${params.bidder}`);
   params.statusCallback ? params.statusCallback(PlaceBidStatus.Encoding) : null;
 
   const hub = await getZnsHubContract(
     params.config.web3Provider,
     params.config.znsHubAddress
   );
   const registrar = await hub.getRegistrarForDomain(params.bid.tokenId);
-  
-  const contract = await getZAuctionContract(params.config.web3Provider, params.config.zAuctionAddress);
-  const paymentToken = await contract.getPaymentTokenForDomain(params.bid.tokenId);
+
+  const contract = await getZAuctionContract(
+    params.config.web3Provider,
+    params.config.zAuctionAddress
+  );
+  const paymentToken = await contract.getPaymentTokenForDomain(
+    params.bid.tokenId
+  );
+  logger.trace(
```

Comment:
Why not just combine with the first log statement?

Reply:
`paymentToken` doesn't exist until this point but I can restructure the message